march_gait_selection/src/march_gait_selection/gait_selection_node.py

```diff
@@ -1,15 +1,17 @@
 import rospy
 from std_srvs.srv import Trigger
 
-from march_shared_resources.srv import ContainsGait, ContainsGaitResponse, SetGaitVersion
+from march_shared_resources.srv import (ContainsGait, ContainsGaitResponse, PossibleGaits, PossibleGaitsResponse,
+                                        SetGaitVersion)
 
 from .gait_selection import GaitSelection
-from .perform_gait_action import PerformGaitAction
 from .state_machine.gait_state_machine import GaitStateMachine
+from .state_machine.state_machine_input import StateMachineInput
 
 NODE_NAME = 'gait_selection'
 GAIT_FILES_MAP_NAME = 'march_gait_files'
-GAIT_DIRECTORY_NAME = 'gait'
+GAIT_DIRECTORY_NAME = 'minimal'
```

Comment:
```suggestion
DEFAULT_GAIT_DIRECTORY_NAME = 'minimal'
```
Also for the other ones